rasa/cli/test.py

```diff
@@ -102,5 +102,12 @@ def test_nlu(args: argparse.Namespace) -> None:
 
 
 def test(args: argparse.Namespace):
+    model = args.model
+
+    # expects a list of models
+    args.model = [model]
```

Comment:
is this to do with the comparison mode or why is that?

Reply:
Yes, due to the comparison test_core expects a list of models.